fuel-tx/src/transaction/types/create.rs

```diff
@@ -396,6 +396,11 @@ mod field {
                     .zip(input.predicate_len().map(bytes::padded_len_usize))
             })
         }
+
+        #[inline(always)]
+        fn inputs_predicate_gas_used_at(&self, idx: usize) -> Option<Word> {
+            self.inputs().get(idx).and_then(|input| input.predicate_gas_used())
```

Comment:
Isn't this supposed to be the location in memory of the predicate? This is returning the value itself

Reply:
removed the function and refactored